R/cts.R

```diff
@@ -163,7 +166,10 @@ cts_convert <- function(query,
       if (verbose) webchem_message("na")
       return(NA)
     }
-    if(verbose) webchem_message("query", query, appendLF = FALSE)
+    if (from == "CAS"){
```

Comment:
Should this be lowercase?
```suggestion
    if (from == "cas"){
```

Reply:
yes, it should. I got confused because I saw the upper case "CAS" in the documentation, but I see now that these arguments are converted to lower case inside the `match.arg`.